### PR TITLE
frege-repl: update sha256

### PR DESCRIPTION
**Pull request by akilegaspi**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by neutric** [comment]
Thanks for your first contribution to Homebrew, @akilegaspi! We'll however need to know why the file has changed before taking further action. Has the fact that the checksum changed been reported upstream yet? If not, please do so, and post a link to the issue here.

**Comment by neutric** [comment]
Filed an [issue](https://github.com/Frege/frege-repl/issues/51) upstream.

**Comment by akilegaspi** [comment]
Hi @neutric, I haven't actually reported it upstream but I just noticed that I wasn't able to install the frege-repl properly so I decided to double check if it were really a file change or not.

**Comment by akilegaspi** [comment]
Hi @neutric, I just checked out the commits since the release of the package and I realized that Frege REPL is actually a snapshot of the master every compile so it is reasonable that the Frege REPL's SHA256 will change because there was a new merge to master on April 18, 2017.

**Comment by neutric** [comment]
And updating the snapshot in place / reusing the tag is not good as it leads to breakage for everyone relying on checksums.

**Comment by neutric** [comment]
Upstream [confirmed the change](https://github.com/Frege/frege-repl/issues/51#issuecomment-298257943).

File: Formula/frege-repl.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class FregeRepl < Formula
   homepage "https://github.com/Frege/frege-repl"
   url "https://github.com/Frege/frege-repl/releases/download/1.4-SNAPSHOT/frege-repl-1.4-SNAPSHOT.zip"
   version "1.4-SNAPSHOT"
-  sha256 "2cf1c2a8f7b64c9d70b21fbfd25b2af3f5e3bebe3662f724afd435d01bddafec"
+  sha256 "2ca5f13bc5efaf8515381e8cdf99b4d4017264a462a30366a873cb54cc4f4640"
```
Comment (neutric) [inline Formula/frege-repl.rb]: Please add `revision 1` on a new line right underneath `sha256` and then this is good to go.

Comment (neutric) [inline Formula/frege-repl.rb]: Please add `revision 1` on a new line right underneath `sha256` and then this is good to go.

**Review by neutric — CHANGES_REQUESTED** [review]

**Comment by fxcoudert** [comment]
Thanks @akilegaspi for your contribution to Homebrew!

**Pull request merged**